src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1404,6 +1737,17 @@ public static void SetTestHook(string property, object value)
                 fieldInfo.SetValue(null, value);
             }
         }
+
+        /// <summary>
+        /// TestImplicitRemotingBatching
```

Comment:
Please add more useful summary and final dot.

Reply:
Will do.